src/ArticleCard.jsx

```diff
@@ -3,34 +3,52 @@ import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import { useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { CardActionArea } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import Article from './Article';
 
 
 
 const ArticleCard = (props) => {
     const { article } = props
+    const [id, setId] = useState(0)
```

Comment:
This doesn't make sense as something you'd hold in state, you're sending the article information on props to the card so that you can render everything, why would you need to hold the id in state here?